docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,89 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-howto"]
+= Update projects to Quarkus 3 automatically
+:extension-status: preview
+include::_attributes.adoc[]
+:categories: core
+:summary: Update projects from Quarkus 2.13 and later to Quarkus 3.
+
+As a developer, you can update your projects from Quarkus 2.13 and later to Quarkus 3 by running one of the update commands.
+These commands use OpenRewrite recipes to automatically update most of your project's dependencies, source code, and documentation.
```

Comment:
_These_ is ambiguous. Could we describe the commands and how you can run them earlier before the steps begin? 

Line 58 has some good information that could be placed next to this line.

`You can run the update command from either the Quarkus CLI, Maven, or JBang.
`

Reply:
Updated. Thanks.